src/engine/sched.c

```diff
@@ -2241,6 +2240,8 @@ sched_run(ABT_sched sched)
 		return;
 	}
 
+	dx->dx_sched_info.si_agg_gap = (vos_get_agg_gap() + 10) * 1000; /* msecs */
```

Comment:
Could you move this to sched_info_init()? Or remove the 'si_agg_gap'? I don't see why we need to duplicate this constant value in sched_info.

Reply:
I can move it into sched_info_init(), but seems not too much difference. I am not sure what "duplicate" you mean. I have ever discussed with you that making such agg_gap as global variable will cause compile dependency trouble.